docs/reference/commandline/trust_signer_remove.md

```diff
@@ -0,0 +1,165 @@
+Root Key:	3cb2228f6561e58f46dbc4cda4fcaff9d5ef22e865a94636f82450d1d2234949
+```
+```bash
+$ docker trust inspect example/trust-demo2
+SIGNED TAG          DIGEST                                                             SIGNERS
+v1                  74d4bfa917d55d53c7df3d2ab20a8d926874d61c3da5ef6de15dd2654fc467c4   alice, bob
+
+List of signers and their KeyIDs:
+
+SIGNER              KEYS
+alice               05e87edcaecb
+bob                 5600f5ab76a2
+
+Administrative keys for example/trust-demo2:
+Repository Key:	ece554f14c9fc399da523a5f4e24fe306a0a6ee1cc79a10e4553d2ab20a8d9268
+Root Key:	3cb2228f6561e58f46dbc4cda4fcaff9d5ef22e865a94636f82450d1d2234949
+```
+Remove `alice` from both images with a single `docker trust signer-remove` command:
+
+```bash
+$ docker trust signer-remove alice example/trust-demo example/trust-demo2
+Enter passphrase for repository key with ID 95b9e55: 
+Successfully removed alice from example/trust-demo
+Enter passphrase for repository key with ID ece554f: 
+Successfully removed alice from example/trust-demo2
+```
+`docker trust inspect` no longer lists `alice` as a valid signer of either `example/trust-demo` or `example/trust-demo2`:
+```bash
+$ docker trust inspect example/trust-demo
+SIGNED TAG          DIGEST                                                             SIGNERS
+v1                  74d4bfa917d55d53c7df3d2ab20a8d926874d61c3da5ef6de15dd2654fc467c4   bob
+
+List of signers and their KeyIDs:
+
+SIGNER              KEYS
+bob                 5600f5ab76a2
+
+Administrative keys for example/trust-demo:
+Repository Key:	ecc457614c9fc399da523a5f4e24fe306a0a6ee1cc79a10e4555b3c6ab02f71e
+Root Key:	3cb2228f6561e58f46dbc4cda4fcaff9d5ef22e865a94636f82450d1d2234949
+```
+```bash
+$ docker trust inspect example/trust-demo2
+SIGNED TAG          DIGEST                                                             SIGNERS
+v1                  74d4bfa917d55d53c7df3d2ab20a8d926874d61c3da5ef6de15dd2654fc467c4   bob
+
+List of signers and their KeyIDs:
+
+SIGNER              KEYS
+bob                 5600f5ab76a2
+
+Administrative keys for example/trust-demo2:
+Repository Key:	ece554f14c9fc399da523a5f4e24fe306a0a6ee1cc79a10e4553d2ab20a8d9268
+Root Key:	3cb2228f6561e58f46dbc4cda4fcaff9d5ef22e865a94636f82450d1d2234949
+```
+
+## Partial success
+
+With `docker trust signer-remove` failure to remove a signer from a repository in the list does not affect the removing of a signer from any other repositories specified:
+
+Having a repository for which the repository is had, `example/authorized`, and having a repository for which alice is not a signer for or the repository key is not had, `example/unauthorized`, `docker trust signer-remove alice example/unauthorized example/authorized` will remove alice print an error example/unauthorized and remove alice from example/authorized:
+
+```bash
+$ docker trust signer-remove alice eiais/test1 example/authorized
+No signer alice for image example/unauthorized
```

Comment:
hm, is this spacing as-is?  I'm wondering if we should (in a followup) add more newlines to make this consistent with `signer-add`?  If so, let's open a tracking issue.